dmutils/content_loader.py

```diff
@@ -413,6 +404,30 @@ def _get_single_question_data(self, form_data):
 
         return {self.id: value if value else None}
 
+    def get_error_messages(self, errors):
+        error_fields = set(errors.keys()) & set(self.form_fields)
+        if not error_fields:
+            return {}
+
+        question_errors = {}
```

Comment:
Multi-questions act as sections when edited – so `ContentQuestion.get_error_messages` is only responsible for one nested question and the order is defined in the `ContentSection.questions`. This dictionary should always end up with one key right now anyway, there's a bit more explanation here: https://github.com/alphagov/digitalmarketplace-utils/pull/223#discussion_r46440837

Another thing about order of errors here is the value we end up with. Since `.form_fields` order is not defined (with fields being a dictionary) for multi-field questions that have multiple errors (eg both max and min price were empty) the error message we end up with is defined by alphabetical order rather than field order (so we could end up with max price error first).